cmd/operator/main.go

```diff
@@ -18,7 +18,7 @@ package main
 
 import (
 	"github.com/spf13/cobra"
-	"k8s.io/klog/v2/klogr"
+	"k8s.io/klog/v2/textlogger"
```

Comment:
Not against it, but curious: why the switch from klogr to textlogger?

Reply:
It's deprecated and the linter flagged it.